server/pulp/server/db/model/repository.py

```diff
@@ -266,11 +267,15 @@ def skipped_result(cls, repo_id, distributor_id, distributor_type_id, started, c
 
         @param result_code: one of the RESULT_* constants in this class
         @type  result_code: str
+
+        @param message: reason for skipping
+        @type message: str
+
         """
 
         r = cls(repo_id, distributor_id, distributor_type_id, started, completed,
                 cls.RESULT_SKIPPED)
-        message = 'Skipped. Nothing changed since last publish'
+        message = _('Skipped: %(reason)s' % {'reason': message})
```

Comment:
The variable replacement is happening before the call to `_()`, but it needs to happen afterwards.